src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicRemoteExecutorTests.cs

```diff
@@ -20,27 +20,43 @@ public class MsQuicRemoteExecutorTests : QuicTestBase
         public MsQuicRemoteExecutorTests()
             : base(null!) { }
 
-        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
-        public void SslKeyLogFile_IsCreatedAndFilled()
+        [ConditionalTheory(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void SslKeyLogFile_IsCreatedAndFilled(bool enabledBySwitch)
```

Comment:
Do we run tests in CI in Release build of the library? If not, we should at least test it locally / manually.

Reply:
Yes, and I locally confirmed that the sslkeylogfile output is well formed